SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Ingestion/Http/RetryableServiceCall.cs

```diff
@@ -1,76 +0,0 @@
-﻿using System;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Microsoft.AppCenter.Ingestion.Http
-{
-    public class RetryableServiceCall : ServiceCallDecorator
```

Comment:
All retry logic in `Ingestion` classes; This is not just refactor; it caused by changing method signatures in public interfaces;